### ISSUE TITLE
Test compilation failure

### ISSUE THREAD

**Issue by bergmark** [andrewthad/haskell-ip#17]
```
Preprocessing test suite 'test' for ip-0.9.2..
Building test suite 'test' for ip-0.9.2..
[...]
test/Test.hs:47:9: error:
    Ambiguous occurrence ‘testCase’
    It could refer to either ‘Test.Framework.Providers.HUnit.testCase’,
                             imported from ‘Test.Framework.Providers.HUnit’ at test/Test.hs:8:46-53
                          or ‘Test.QuickCheck.Property.testCase’,
                             imported from ‘Test.QuickCheck.Property’ at test/Test.hs:11:63-72
   |
47 |       , testCase "Decode an IP" testIPv4Decode
   |         ^^^^^^^^

```

**Comment by andrewthad** [comment]
This was fixed in f480474db568b72b01c76391aa4a95be6d724a5d. I'll release it to hackage.

**Comment by andrewthad** [comment]
Released in 0.9.3.
